### PR TITLE
More verbose logging for translation

### PR DESCRIPTION
**Pull request by aurimasv**
Would make it easier to make sure that users have up-to-date translators e,g, https://forums.zotero.org/discussion/47939/import-keywords-containing-several-words/

### CODE REVIEW

File: chrome/content/zotero/xpcom/translation/translate.js [hunk 1]
```diff
@@ -1528,7 +1535,8 @@ Zotero.Translate.Base.prototype = {
 		this._aborted = false;
 		this.saveQueue = [];
 		
-		Zotero.debug("Translate: Parsing code for "+translator.label, 4);
+		Zotero.debug("Translate: Parsing code for " + translator.label
+			+ " -- " + translator.translatorID + " -- " + translator.lastUpdated, 4);
```
Comment (dstillman) [inline chrome/content/zotero/xpcom/translation/translate.js]: Let's go with this:
```
		Zotero.debug("Translate: Parsing code for " + translator.label + " "
			+ "(" + translator.translatorID + ", " + translator.lastUpdated + ")", 4);
```

Comment (aurimasv) [inline chrome/content/zotero/xpcom/translation/translate.js]: I tried that, but it looks really dense in the debug output. Personally, I find this style more pleasant. Try it out if you have time.

Comment (dstillman) [inline chrome/content/zotero/xpcom/translation/translate.js]: I did. I hate it.

Comment (dstillman) [inline chrome/content/zotero/xpcom/translation/translate.js]: And the extra info really is parenthetical. Generally people will have the right translator and the right version.

**Comment by aurimasv** [comment]
Changed

**Comment by dstillman** [comment]
Great, thanks.

**Pull request merged**